src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Strings.Designer.cs

```diff
@@ -8,34 +8,35 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace NuGet.Build.Tasks.Pack {
+namespace NuGet.Build.Tasks.Pack.Library {
```

Comment:
Using ``Run Custom Tool`` through VS automatically set this. Maybe the project needs to override the default namespace. I'll take a look.